### ISSUE TITLE
Move from Docsify

### ISSUE THREAD

**Issue by tachyons** [CircuitVerse/CircuitVerseDocs#383]
We are currently using docsify for documentation, But it is not SEO friendly and is causing issues in discoverability . We need to find another tool which works well for this

Upstream issue: https://github.com/docsifyjs/docsify/issues/136

**Comment by salmoneatenbybear** [comment]
@tachyons how about Docusaurus ?

It's optimized for SEO, 

Also supports Markdown, pretty easy setup, and includes built-in features for versioning and localization.

**Comment by tachyons** [comment]
That looks good

**Comment by salmoneatenbybear** [comment]
Hi, @tachyons I have migrated the documentation site from Docsify to Docusaurus. 

This is the GitHub repo for the new Docusaurus powered CircuitVerse Documentation : 

**https://github.com/salmoneatenbybear/circuitverse-docs**

This is the live link of the website deployment on netlify : 

**https://673b859c3974ec2d71c0375e--circuitverse-docs.netlify.app/**

**Comment by ThatDeparted2061** [comment]
What are the updates on the shift to docusaurus

**Comment by salmoneatenbybear** [comment]
@ThatDeparted2061 

https://github.com/salmoneatenbybear/circuitverse-docs

Made the new site.

**Comment by ThatDeparted2061** [comment]
Yes I am aware of that, what are the updates on integrating it with the codebase

**Comment by salmoneatenbybear** [comment]
> Yes I am aware of that, what are the updates on integrating it with the codebase 
> 

That needs to be processed further by documentation site current maintainers.
